.github/workflows/build-test-package.yml

```diff
@@ -157,8 +56,12 @@ jobs:
 
     - name: 'Build 🐍 Python 📦 package'
       run: |
-        export ITK_PACKAGE_VERSION=${{ matrix.itk-python-git-tag }}
-        ./dockcross-manylinux-download-cache-and-build-module-wheels.sh cp${{ matrix.python-version }}
+        export ITK_PACKAGE_VERSION=${{ env.itk-wheel-tag }}
+        for tarball in "" "-manylinux2014"; do
+          rm -rf ITKPythonPackage
+          export TARBALL_SPECIALIZATION=${tarball}
+          ./dockcross-manylinux-download-cache-and-build-module-wheels.sh cp${{ matrix.python-version }}
```

Comment:
It seems that "-manylinux2014" tarballs are failing:

```
-- Build files have been written to: /work/_cmake_test_compile/build
CMake Error at _skbuild/linux-x86_64-3.8/cmake-build/CMakeFiles/3.23.1/CMakeSystem.cmake:6 (include):
  include could not find requested file:

    /opt/rh/gcc-toolset-11/root/lib/Toolchain.cmake
Call Stack (most recent call first):
  CMakeLists.txt:2 (project)


CMake Error at CMakeLists.txt:2 (project):
  The CMAKE_C_COMPILER:

    /opt/rh/gcc-toolset-11/root/bin//gcc

  is not a full path to an existing compiler tool.
```

It is not immediately clear to me why `Toolchain.cmake` and `gcc` cannot be found. Note that we are using the `ubuntu-20.04` runner in ITKSplitComponents to build `_2_28` and `-manylinux2014` tarballs without errors: [https://github.com/InsightSoftwareConsortium/ITKSplitComponents/blob/master/.github/workflows/build-test-package.yml#L59](https://github.com/InsightSoftwareConsortium/ITKSplitComponents/blob/master/.github/workflows/build-test-package.yml#L59)

Depending on the priority level of this commit perhaps `-manylinux2014` could be dropped from this PR for now pending further investigation?

Reply:
Hmm, I wonder if https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/202 is not cleaning something required relative to the compiler reference. Yes, let's skip `-manylinux2014` for now.

Reply:
I am rerunning [ITKSplitComponents CI](https://github.com/InsightSoftwareConsortium/ITKSplitComponents/actions/runs/2981185053) for cross referencing.

Reply:
ITKSplitComponents CI is now failing; I agree that this looks like an issue in ITKPythonPackage. Will enter an issue to track there.